<launch>

	<node name="jetbot_camera" pkg="jetbot_ros" type="jetbot_camera"/> 
<!--	<node name="find_keypoints_node" pkg="stright_moving" type="find_keypoints_node" /> -->

</launch>
